Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -62,15 +121,15 @@ protected override Size MeasureOverride(Size availableSize)
 
                 if (parentMeasure.U > currentMeasure.U + lineMeasure.U)
```

Comment:
Does this need to be updated to account for the spacing as well?

Reply:
Is that the cause of this issue or something else?

![image](https://user-images.githubusercontent.com/24302614/32824483-07e6f636-c996-11e7-918c-727d86ce5d16.png)
(The image isn't cut on the right, the panel is doing that as it should have wrapped earlier as I was bringing in the right edge.)

If you collapse the edge, it looks like it's missing about the size of the horizontal alignment in the calculation if there's enough space before wrapping.

Reply:
After adding `if (parentMeasure.U > currentMeasure.U + lineMeasure.U + _uSpacing)` solves this issue

Reply:
![1](https://user-images.githubusercontent.com/6191377/32826948-81d66d8e-ca3e-11e7-8ad3-1b93e256f2a3.PNG)
There is also another issue in the vertical items

Reply:
Both issues should have been fixed by the same calculation change in the latest commit.